tests/dune

```diff
@@ -1,5 +1,4 @@
 (library
  (name wasmtime_tests)
- (inline_tests)
```

Comment:
Why is this line removed? Would the tests still be run as part of the CI/"dune runtest" after this change?